common/migration/1685136338879-AddConversationIdToSubcourse.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddConversationIdToSubcourse1685136338879 implements MigrationInterface {
+    name = 'AddConversationIdToSubcourse1685136338879';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "subcourse" ALTER COLUMN "conversationId"`);
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "subcourse" ALTER COLUMN "conversationId"`);
+    }
+}
```

Comment:
Can you please recreate this migration, I'm a bit scared of it honestly. Because now it does nothing. 
When did you bring the change with the `conversationId` in? In the last PR or in this one?

Reply:
in this PR

Reply:
Then it should've been something like
```sql
ALTER TABLE "subcourse" ADD "conversationId" character varying
```
and
```sql
ALTER TABLE "subcourse" DROP "conversationId"
```